samples/interactive-tutorials/product/setup_product/products_create_gcs_bucket.py

```diff
@@ -23,7 +24,9 @@
 bucket_name = f"{project_id}_products_{timestamp_}"
 
 create_bucket(bucket_name)
-upload_blob(bucket_name, "../resources/products.json")
-upload_blob(bucket_name, "../resources/products_some_invalid.json")
+
+path_to_resources_dir = f"{Path.home()}/cloudshell_open/python-retail/samples/interactive-tutorials/resources/"
+upload_blob(bucket_name, path_to_resources_dir + "products.json")
```

Comment:
You are using pathlib in this script, which excellent; however here you are not using all the strength that this design decision gives you (for example path concatenation with "/" iirc), and instead using string conatenation. Why is that?

Reply:
fixed